sdk/python/tests/compiler/testdata/loop_with_numeric.yaml

```diff
@@ -0,0 +1,157 @@
+                    pipelines.kubeflow.org/pipelinename: ''
+                    pipelines.kubeflow.org/generation: ''
+                    pipelines.kubeflow.org/cache_enabled: "true"
+                  annotations:
+                    pipelines.kubeflow.org/component_spec_digest: '{"name": "my-in-coop1",
+                      "outputs": [], "version": "my-in-coop1@sha256=42f0c6a60c7e435441b8afaeb382a771a9741fe3aabb203748fdbd72b25f1628"}'
+                    tekton.dev/template: ''
+              timeout: 525600m
+            - name: my-pipeline-for-loop-4
+              params:
+              - name: from
+                value: '1'
+              - name: my_pipe_param
+                value: $(params.my_pipe_param)
+              - name: to
+                value: '2'
+              taskSpec:
+                apiVersion: custom.tekton.dev/v1alpha1
+                kind: PipelineLoop
+                spec:
+                  pipelineSpec:
+                    params:
+                    - name: loop-item-param-3
+                      type: string
+                    - name: my_pipe_param
+                      type: string
+                    tasks:
+                    - name: my-in-coop1-2
+                      params:
+                      - name: loop-item-param-3
+                        value: $(params.loop-item-param-3)
+                      - name: my_pipe_param
+                        value: $(params.my_pipe_param)
+                      taskSpec:
+                        steps:
+                        - name: main
+                          args:
+                          - |
+                            set -e
+                            echo op1 "$0" "$1"
+                          - $(inputs.params.loop-item-param-3)
+                          - $(inputs.params.my_pipe_param)
+                          command:
+                          - sh
+                          - -c
+                          image: library/bash:4.4.23
+                        params:
+                        - name: loop-item-param-3
+                          type: string
+                        - name: my_pipe_param
+                          type: string
+                        metadata:
+                          labels:
+                            pipelines.kubeflow.org/pipelinename: ''
+                            pipelines.kubeflow.org/generation: ''
+                            pipelines.kubeflow.org/cache_enabled: "true"
+                          annotations:
+                            pipelines.kubeflow.org/component_spec_digest: '{"name":
+                              "my-in-coop1", "outputs": [], "version": "my-in-coop1@sha256=42f0c6a60c7e435441b8afaeb382a771a9741fe3aabb203748fdbd72b25f1628"}'
+                            tekton.dev/template: ''
+                      timeout: 525600m
+                  iterateNumeric: loop-item-param-3
+          iterateNumeric: loop-item-param-1
+  timeout: 525600m
```

Comment:
Did you run it, by any chance?

Reply:
Yes I did. It works fine for this example.